Dockerfile

```
@@ -0,0 +1,9 @@
+FROM quay.io/thoth-station/s2i-thoth-ubi8-py36
+
+USER 0
+
+RUN sudo dnf install cloc -y
```

```suggestion
RUN dnf install cloc -y
```
user 0 is already set, it is running as root. 
sudo might causes.